RELEASING.md

```diff
@@ -0,0 +1,78 @@
+# Release Workflow
+
+Version information is maintained in the `gradle.properties` file with the
+`prefab.version` and `prefab.qualifier` properties. The version number should be
+in the `<major>.<minor>.<patch>` format and the qualifier is either empty (for a
+final release) or should be in the `-<RELEASE_TYPE><INCREMENT>` format. The
+release type should be alpha, beta, or rc, and the increment begins at 1 and
+increases for each additional release of the same version and type. For example,
+the second release candidate of 1.0.0 should have the following entries in
+gradle.properties:
+
+```properties
+prefab.version = 1.0.0
+prefab.qualifier = -rc2
+```
+
+Version 1.0.0 final would have the following:
+
+```properties
+prefab.version = 1.0.0
+prefab.qualifier =
+```
+
+The concatenated `<VERSION_NUMBER><QUALIFIER>` will be referred to as
+`<VERSION>` for the remained of this doc.
+
+To create a new release of Prefab:
+
+1. Checkout and update the master branch, create a branch to prepare the pull
+   request.
+
+   ```bash
+   git checkout master
```

Comment:
This isn't necessarily what you'd want to release from, right?

Reply:
Added a note clarifying that you might want to use a different branch point.